test/rulesets/LinearAlgebra/dense.jl

```diff
@@ -115,13 +115,21 @@
         rrule_test(tr, randn(), (randn(N, N), randn(N, N)))
     end
     @testset "*" begin
-        dims = [3,4,5]
-        for n in dims, m in dims, p in dims
-            n > 3 && n == m == p && continue  # don't need to test square case multiple times
-            A = randn(m, n)
-            B = randn(n, p)
-            Ȳ = randn(m, p)
-            rrule_test(*, Ȳ, (A, randn(m, n)), (B, randn(n, p)))
+        @testset "Matrix-Matrix" begin
```

Comment:
Out of scope perhaps but are there tests for Vector*Matrix anywhere? `dot` is tested but I don't think `*` is

Reply:
I am not sure we actually have a matric vector multiply `rrule` defined.

Reply:
cool I would be keen to tackle this